### DISCUSSION TITLE
If a section has more entries than `max_input_vars` php setting, cannot check all

### DISCUSSION THREAD

**Discussion by GaryReckard** [trendyminds/isolate#39] [Q&A]
I'm working on a site that has a large number of entries. With this plugin installed, I am editing a users Isolate settings for a particular section. If I choose to check-all, and submit, the resulting page only shows checkboxes for the first X entries, where X is around the same as the php setting `max_input_vars`.

This isn't necessarily a bug with your plugin, but seems like it might be an issue others run into. The interface shows no sign that this issue happens, unless you happen to scroll down and see all the unchecked entries.

One stop-gap solution could be to detect the `max_input_vars` value, and if the number of entries displayed is greater than that, show a warning at the top of the page.

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
